charts/cargohold/templates/configmap.yaml

```diff
@@ -41,7 +41,14 @@ data:
 
     settings:
       {{- with .Values.config.settings }}
-      maxTotalBytes: 5368709120
-      downloadLinkTTL: 300      # Seconds
+      {{- if .maxTotalBytes }}
+      maxTotalBytes: {{ .maxTotalBytes }}
+      {{- end }}
```

Comment:
This causes this change in the configmap:
before:
```
maxTotalBytes: 5368709120
```
after
```
maxTotalBytes: 5.36870912e+09
```
This has no effect on the value read by the options parser howerver